### DISCUSSION TITLE
[Icon Request] check-square

### DISCUSSION THREAD

**Discussion by thaiphan** [tailwindlabs/heroicons#459] [Ideas]
Hey guys, would it be possible to add a square checkbox icon (both `checked` and `unchecked` states if possible)? There's a circle-check but I'd also appreciate a square version!

Thanks for your time!

**Comment by youfoundron** [comment]
This!

**Comment by theoephraim** [comment]
I'd expand the request to add a square/box variation on a few icons - things like plus, minus, check, x, etc...

Probably not necessary for every one, but many of the icons where we have `thing-circle` could benefit from a square version.

The same square from `code-bracket-square` could be used as the box.

**Comment by grahamplace** [comment]
Strong agree, it should be possible to make a simple "checklist" with just these icons (checked/unchecked squares)

**Comment by rahmat-ullah-008** [comment]
I was a bit surprised to see that the square-check icon has not been added yet.
